.github/workflows/vuln-scans.yml

```diff
@@ -206,24 +224,31 @@ jobs:
 
   snyk-docker-scan:
     runs-on: ubuntu-latest
-    needs: docker-otelcol
+    needs: [docker-otelcol, check-snyk-token]
+    if: ${{ needs.check-snyk-token.outputs.has-snyk-token == 'true' }}
     strategy:
       matrix:
         ARCH: [ "amd64", "arm64" ]
       fail-fast: false
     steps:
       - uses: actions/checkout@v4
+      - uses: docker/setup-qemu-action@v3
+        if: ${{ matrix.ARCH != 'amd64' }}
+        with:
+          platforms: ${{ matrix.ARCH }}
+          image: tonistiigi/binfmt:qemu-v7.0.0
```

Comment:
Is this image something that we can rely will be around for the long haul?

Reply:
> Is this image something that we can rely will be around for the long haul?

I hope so.  It is the default image for this action (https://github.com/marketplace/actions/docker-setup-qemu#inputs).  We're just using a newer tag (`latest` is currently v6.2.0).